features/bootstrap/FeatureContext.php

```diff
@@ -355,6 +421,66 @@ public function thereIsDummyObjectsWithDummyBoolean($nb, $bool)
     }
 
     /**
+     * @Given there is :nb embedded dummy objects with embeddedDummy.dummyBoolean :bool
+     */
+    public function thereIsDummyObjectsWithEmbeddedDummyBoolean($nb, $bool)
+    {
+        if (in_array($bool, ['true', '1', 1], true)) {
```

Comment:
Note that I'm sure we talked about `FILTER_VALIDATE_BOOLEAN` when working on the boolean filter, IIRC it was with @teohhanhui.